app/react-native/src/preview/index.js

```diff
@@ -55,8 +57,7 @@ export default class Preview {
       let webUrl = null;
       let channel = addons.getChannel();
       if (params.resetStorybook || !channel) {
-        const host = params.host || 'localhost';
-
+        const host = params.host || parse(NativeModules.SourceCode.scriptURL).hostname;
         const port = params.port !== false ? `:${params.port || 7007}` : '';
```

Comment:
Is this the port for the ReactNative packager? Or where storybooks web previewed will be binding to?